README.md

```diff
@@ -6,7 +6,16 @@
 ![GO tests](https://github.com/mongodb/mongocli/workflows/GO%20tests/badge.svg)
 ![golangci-lint](https://github.com/mongodb/mongocli/workflows/golangci-lint/badge.svg)
 
-`mongocli` is a tool for managing your MongoDB cloud services
+The MongoDB CLI is a modern command line interface that enables you to manage your MongoDB services from the terminal.
```

Comment:
[q] Should it be `MongoCLI` instead of `MongoDB CLI`? I got bit confused by reading `MongoDB CLI`.

Reply:
Official name MongoDB CLI, `mongocli` for short